packages/ui/src/forum/components/threads/ThreadListItem.tsx

```diff
@@ -35,10 +36,12 @@ export const ThreadListItem = ({ thread, isArchive }: ThreadListItemProps) => {
     >
       {thread.isSticky && <ThreadPinIcon />}
 
-      <Thread>
-        <TextBig bold>{thread.title}</TextBig>
-        {thread.tags.length > 0 && <ThreadTags tags={thread.tags} />}
-      </Thread>
+      <Tooltip tooltipText={thread.title}>
+        <Thread>
+          <TextBig bold>{thread.title}</TextBig>
```

Comment:
> "Radosław" add this line of code to the title component for truncation:
> truncateLines={2}

Reply:
![image](https://user-images.githubusercontent.com/69617937/197877962-5786644b-f1f1-463f-b24a-1c397649f502.png)
This is the CSS which is for truncated features (I just checked that it already fixed)

Reply:
`FullDots` will truncate first line only. `truncateLines` props on all `Text` components allows to pick number of line which will be truncated. `truncateLines={1}` should work as well